wie-geht-studium/medienplattformen_mathe.tex

```diff
@@ -0,0 +1,16 @@
+\subsection{Mathematische Medienplattform -- MaMpf}
+\label{mampf}
+
+Bücher sind nicht für alle das geeignete Lernmittel, daher existiert seit dem Wintersemester 2016 \emph{MaMpf}\footnote{Erreichbar unter\url{https://mampf.mathi.uni-heidelberg.de/}}, die Mathematische Medienplattform. Initiiert wurde sie von Dr. Denis Vogel. Geboten wird ein vielfältiges Angebot, welches beim Lernen sehr nützlich sein kann.
+
+\subsubsection{KaViaR -- Kameraloses Videoaufzeichnungssystem zur Rekapitulationsunterstüzung}
+\emph{KaViaR} ist das Herzstück von MaMpf, kurz gesprochen handelt es sich um eine Videoplattform speziell für Mathevideos. Besonders praktisch ist, dass ihr auch Videos von vergangenen Vorlesungen zugreifen könnt. Zurzeit ist das Projekt  Passwort geschützt, dies erfahrt ihr in eurer ersten „MaMpf“--Vorlesung oder fragt eine Studierende aus höheren Semester.
+
+\subsubsection{KeKs -- Kompetenzerweiterendes Kurzfragensystem}
+In vielen Prüfungen der ersten Semester werden eure Klausuren einen Multiple--Choice--Teil haben.  Darauf vorbereiten könnt ihr euch mit über 5000 Fragen mit \emph{KeKs}, dem Kompetenzerweiterendes Kurzfragensystem. Zum Teil werden die Antworten nicht nur mit „wahr“ oder „falsch“, sondern auch durch Erklärvideos verständlich vermittelt.
```

Comment:
In dem Satz muss es heißen "dem Kompetenzerweiterende**n**"

Und es fehlt ein Wort in "Zum Teil werden die Antworten nicht nur mit „wahr“ oder „falsch“,"